src/index.html

```diff
@@ -10,7 +10,36 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="search">
+    <form
+    action="Get"
+    class="search__form"
+    data-qa="big">
```

Comment:
```suggestion
    <form
      action="Get"
      class="search__form"
      data-qa="big"
     >
```
pay attention to your code style

Reply:
check it everywhere